storage/innobase/include/sync0policy.ic

```diff
@@ -31,13 +31,29 @@ std::string GenericPolicy<Mutex>::to_string() const
 	return(sync_mutex_to_string(get_id(), sync_file_created_get(this)));
 }
 
+/** BlockMutexPolicy should be renamed as AggregateMutexStatsPolcy. This Policy
```

Comment:
Grepping suggests about a dozen places to be fixed if we actually proceeded with the rename. Maybe it's not too bad for maintenance?

Reply:
Ok, If you think there is no merge trouble, I can ahead.

IMO too, this code doesn't change much. Meaning not many new mutexes will use this BlockMutexPolicy. So we can rename this.